challenge_03_loops-conditionals/loops-conditionals/loops/exercise_00.js

```diff
@@ -0,0 +1,15 @@
+/*
+    1. Create a for loop that will print out all the even numbers between 10 and 40.
+    
+    2. Create a for loop that will print out all the odd numbers between 10 and 40.
+*/
+
+//exercise 1.
+for (let i = 10; i <= 40; i += 2) {
```

Comment:
You should take in account what `i` variable is storing. Is this variable referring to any index in this case, or a number?

⚠️ FIX: rename properly `i` variable

These loops should iterate over all numbers in between 10 and 40, printing only the odd/even ones, depending on its own logic.

⚠️ FIX: iterate over all numbers and print only the odd/even ones